uvicgse/project-virtual-team-2
# Add `git mv` functionality

### Things to test

* click the 'Move' button located in the footer
* Try moving files/directories into other directories
* Try renaming files, click 'Enter' and see the results reflecting in the staging section

### Notes on design

https://docs.google.com/document/d/16GXaGN1NLkuymykhNoTHSvod5zixT_bPAyJO4RVEggY/edit#
